app/views/contexts/section_preview_context.py

```diff
@@ -0,0 +1,94 @@
+from functools import cached_property
+from typing import Mapping, Optional
+
+from app.data_models import AnswerStore, ListStore, ProgressStore
+from app.data_models.metadata_proxy import MetadataProxy
+from app.questionnaire import QuestionnaireSchema
+from app.questionnaire.location import Location
+from app.utilities import safe_content
+
+from .context import Context
+from .summary import PreviewGroup
+
+
+class SectionPreviewContext(Context):
+    def __init__(
+        self,
+        language: str,
+        schema: QuestionnaireSchema,
+        answer_store: AnswerStore,
+        list_store: ListStore,
+        progress_store: ProgressStore,
+        metadata: Optional[MetadataProxy],
+        response_metadata: Mapping,
+        current_location: Location,
+    ):
+        super().__init__(
+            language,
+            schema,
+            answer_store,
+            list_store,
+            progress_store,
+            metadata,
+            response_metadata,
+        )
+        self.current_location = current_location
+
+    def __call__(self, return_to: Optional[str] = "section-summary") -> Mapping:
+        summary = self._build_summary()
+        title_for_location = self._title_for_location()
+        title = (
+            self._placeholder_renderer.render_placeholder(
+                title_for_location, self.current_location.list_item_id
+            )
+            if isinstance(title_for_location, dict)
+            else title_for_location
+        )
+
+        page_title = self.get_page_title(title_for_location)
+
+        return {
+            "summary": {
+                "title": title,
+                "page_title": page_title,
+                "summary_type": "SectionSummary",
```

Comment:
Are `summary_type` and `answers_are_editable` needed here?

Reply:
Not needed, removed, thanks!